pkg/v1/config/clientconfig.go

```diff
@@ -103,6 +105,7 @@ const (
 var (
 	DefaultCliFeatureFlags = map[string]bool{
 		FeatureContextAwareCLIForPlugins:                      common.ContextAwareDiscoveryEnabled(),
+		FeatureContextCommand:                                 false,
```

Comment:
nit: 
Not sure how we plan to use the flag, but should we add this feature flag to the config?
The option is definitely there, but so it not advertising it at all.
If the purpose of the flag is to use it for internal/testing, but there is no intention of making it deactivate-able when we turn it on, then there is little reason to introduce it in the config.

Reply:
I don't feel strongly one way or the other, especially if we don't plan to retain the use of this flag once the feature is eventually lit up.

Reply:
I am ok with retain the reference in the config on the assumption that do not plan to use it once the functionality is lit up.